src/main/resources/resources/JahiaSitemap_fr.properties

```diff
@@ -1,3 +1,3 @@
-jseomix_sitemapResource=SEO Sitemap
-jseomix_sitemapResource.noIndex=No Index
-jseomix_sitemapResource.createSitemap=Create separate sitemap for this resource
+jseomix_sitemapResource=SEO sitemap
+jseomix_sitemapResource.noIndex=Pas d?index (par d�faut toutes les pages sont en sitemap)
```

Comment:
Not entirely sure but I have a strange feeling about that "?" in "d?index" :)

Reply:
I believe this is a new french accent character, but I changed just for you (thanks for the catch)